tests/test_cmab.py

```
@@ -649,26 +692,37 @@ def test_update(
     # Handle multi-objective rewards for MO models
 
     for_update_kwargs = {"actions": actions_to_update, "rewards": reward_data}
-    if any(isinstance(model, BaseCmabZoomingModel) for model in cmab.actions.values()):
+    if any(isinstance(model, BaseQuantitativeBayesianNeuralNetwork) for model in cmab.actions.values()):
         quantity_data = np.random.random(size=n_samples).tolist()
         quantity_data = [
             q if isinstance(cmab.actions[action], QuantitativeModel) else None
             for q, action in zip(quantity_data, actions_to_update)
```

Ruff (0.15.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)